src/scene/scene.js

```diff
@@ -150,6 +150,13 @@ class Scene extends EventHandler {
      */
     sky = null;
 
+    /**
+     * Use physically based units for cameras and lights.
+     *
+     * @type {boolean}
+     */
+    physicalUnits = true;
```

Comment:
I know this has already been renamed once, but probably `physicalLightUnits` would make more sense?

Reply:
It's also units for cameras, they are not light units surely?

Reply:
yeah right... physical units just sounds like a physics engine setting! :D

Reply:
maybe physicallyBasedRendering